### PR TITLE
headrags will not stop bullets anymore. FOr whatever reason, they did.

### PR DESCRIPTION
**Pull request by KoishiVibe**
# summary
- Weapon SPecialist Head-Rag no longer has armor rating or decapacitation protection inherited from /obj/item/clothing/head/helmet

### CODE REVIEW

File: code/modules/clothing/head/helmet.dm [hunk 1]
```diff
@@ -1214,7 +1214,7 @@ GLOBAL_LIST_INIT(allowed_helmet_items, list(
 
 //head rag
 
-/obj/item/clothing/head/helmet/specrag
+/obj/item/clothing/head/specrag
```
Comment (AndroBetel) [inline code/modules/clothing/head/helmet.dm:1217]: > headrags will not stop bullets anymore.

image

it still has armor defines and they still work.....

**Review by AndroBetel — CHANGES_REQUESTED** [review]

File: code/modules/clothing/head/helmet.dm (hunk 1 above)
Comment (KoishiVibe) [reply code/modules/clothing/head/helmet.dm:1217]: ok what the hell- huh. Weird. Alright, that's fixed. I wonder what happened to those then...? Whatever. We good!

File: code/modules/clothing/head/helmet.dm [hunk 2]
```diff
@@ -1220,26 +1220,26 @@ GLOBAL_LIST_INIT(allowed_helmet_items, list(
...
+	armor_laser = CLOTHING_ARMOR_NONE
 	armor_energy = CLOTHING_ARMOR_NONE
-	armor_bomb = CLOTHING_ARMOR_MEDIUMLOW
-	armor_bio = CLOTHING_ARMOR_MEDIUMLOW
-	armor_rad = CLOTHING_ARMOR_MEDIUMLOW
-	armor_internaldamage = CLOTHING_ARMOR_MEDIUM
+	armor_bomb = CLOTHING_ARMOR_NONE
+	armor_bio = CLOTHING_ARMOR_NONE
+	armor_rad = CLOTHING_ARMOR_NONE
+	armor_internaldamage = CLOTHING_ARMOR_NONE
```
Comment (AndroBetel) [inline code/modules/clothing/head/helmet.dm:1235]: i'm preeeetty sure `/obj/item/clothing/head` type doesnt have any armor values defined so you should just straight up delete them

**Review by AndroBetel — CHANGES_REQUESTED** [review]
you should also probably move this somewhere it should be since it's no longer a helmet

**Comment by KoishiVibe** [comment]
> you should also probably move this somewhere it should be since it's no longer a helmet

don't wanna modify any... oh god does this break gives or-fucksake ill do that in a bit

File: code/modules/clothing/head/helmet.dm (hunk 2 above)
Comment (KoishiVibe) [reply code/modules/clothing/head/helmet.dm:1235]: Done.

**Comment by morrowwolf** [comment]
Waiting on:
"You should also probably move this somewhere it should be since it's no longer a helmet."

Marking draft.

**Comment by KoishiVibe** [comment]
> Waiting on: "You should also probably move this somewhere it should be since it's no longer a helmet."
> 
> Marking draft.

You mean remove from loadouts that have it?

**Comment by KoishiVibe** [comment]
> Waiting on: "You should also probably move this somewhere it should be since it's no longer a helmet."
> 
> Marking draft.

Ah I see. Maybe? There are hats in the helmet section though. Should I shuffle all of those or...?

**Comment by morrowwolf** [comment]
> > Waiting on: "You should also probably move this somewhere it should be since it's no longer a helmet."
> > Marking draft.
> 
> Ah I see. Maybe? There are hats in the helmet section though. Should I shuffle all of those or...?

Just gotta do this one, if we fixed every inconsistency in CM we'd be here all year.

**Comment by AndroBetel** [comment]
author decided to take a break from ss13, if they return they can contact any maintainer to have their PRs reopened

**Pull request closed without merging**